x-pack/plugins/dataset_quality/public/components/dataset_quality/dataset_quality.tsx

```diff
@@ -26,6 +28,10 @@ export const createDatasetQuality = ({ core, plugins }: CreateDatasetQualityArgs
   return ({ controller }: DatasetQualityProps) => {
     const KibanaContextProviderForPlugin = useKibanaContextForPluginProvider(core, plugins);
 
+    const dataStreamStatsClient = new DataStreamsStatsService().start({
```

Comment:
Why are initialising the service here?
It's already initialised in [here](https://github.com/elastic/kibana/blob/52f35fca662bffe99ee3ba431f7d1e94a50ac903/x-pack/plugins/dataset_quality/public/controller/create_controller.ts#L38).